pennylane_sf/gbs.py

```diff
@@ -66,28 +75,29 @@ class StrawberryFieldsGBS(StrawberryFieldsSimulator):
 
     _capabilities = {"model": "cv", "provides_jacobian": True}
 
-    def __init__(self, wires, *, analytic=True, cutoff_dim, shots=1000):
+    def __init__(
+        self, wires, *, analytic=True, cutoff_dim, shots=1000, use_cache=False, samples=None
+    ):
         super().__init__(wires, analytic=analytic, shots=shots)
         self.cutoff = cutoff_dim
+        self.use_cache = use_cache
+        self.samples = samples
+
         self._params = None
         self._WAW = None
+        self.Z_inv = None
 
     @staticmethod
-    def calculate_WAW(params, A, n_mean):
+    def calculate_WAW(params, A):
```

Comment:
Is this change for the purpose of this PR, or a more general change that is being included here for convenience? In other words, what's the reason for removing `n_mean` as an argument?